src/internal/kopia/upload.go

```diff
@@ -73,7 +73,10 @@ func (rw *backupStreamReader) Close() error {
 	rw.combined = nil
 
 	for _, r := range rw.readers {
-		r.Close()
+		err := r.Close()
```

Comment:
@ashmrtn I think this should be harmless, but could you please review this change? :)